app/src/main/res/values/strings.xml

```diff
@@ -6,15 +6,15 @@
     <string name="last_name">Last Name</string>
     <string name="login_title">Login</string>
     <string name="create_act">Create Account</string>
-    <string name="login_password">Enter your password here</string>
+    <string name="login_password">********</string>
```

Comment:
Is this part of password hint?

Reply:
Yeah bro, it's like a demo entry, to make it look a little modern, if you want I'll change it back

Reply:
No, it's fine. Just asked to know if they are part of Password hint. Seems good👍

Reply:
Thanks bro